### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by zee-pk**
Updated readme file.

### CODE REVIEW

**Review by HamzaIbnFarooq — APPROVED** [review]
:+1: 
Do we need to explain dependency of `openedx-caliper-tracking` on `track` and `eventtracking` ?

File: README.rst [hunk 1]
```diff
@@ -40,23 +40,31 @@ these files should be located at ``/edx/app/edxapp/`` directory:
 Location of Transformed Logs
 ****************************
 
-Transformed event logs can be found in ``/edx/var/logs/tracking/tracking.log`` file.
+Transformed events are logged using **'logging.handlers.SysLogHandler'** with **'facility: local2'**.
+
+We need to create output files manually and set appropriate permissions for syslog user, please follow the steps below:
```
Comment (HamzaIbnFarooq) [inline README.rst]: what if we use `To do so, please follow the steps below`

**Review by danialmalik — APPROVED** [review]

**Review by imhassantariq — APPROVED** [review]

**Review by mumarkhan999 — APPROVED** [review]
:+1:

**Review by asamolion — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (zee-pk) [reply README.rst]: Done, please review again

**Review by HamzaIbnFarooq — APPROVED** [review]
:+1:

**Pull request merged**